learn/generation/rag-for-hybrid/rag-for-hybrid-search.ipynb

```diff
@@ -0,0 +1,2569 @@
+{
```

Comment:
<p>Points to local mac path</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Oh, I left this because users were instructed to download the PDFs locally. I have them in our repo in case users want to download them from us instead of Arxiv.</p><p> </p><p>Do you think they should be read from the repo itself? Since this is just a demo, I was thinking ppl would come w/their own PDFs when trying it out </p>